### PR TITLE
[IOTDB-1416] Maintanance command support in cluster mode

### PR DESCRIPTION
**Pull request by ijihang**
such as CREATE SNAPSHOT FOR SCHEMA, CLEAR CACHE  MERGE,Maintanance command support in cluster mode

### CODE REVIEW

**Review by mychaow — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java [hunk 1]
```diff
@@ -436,6 +448,9 @@ public static PhysicalPlan create(ByteBuffer buffer) throws IOException, Illegal
     DROP_TRIGGER,
     START_TRIGGER,
     STOP_TRIGGER,
+    MERGE,
+    CREATE_SNAPSHOT,
+    CLEARCACHE,
     CREATE_CONTINUOUS_QUERY,
     DROP_CONTINUOUS_QUERY,
     SHOW_CONTINUOUS_QUERIES
```
Comment (neuyilan) [inline server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java:456]:
```
MERGE,
CREATE_SNAPSHOT,
CLEARCACHE,
```
Move above codes to the end.

**Review by neuyilan — CHANGES_REQUESTED** [review]

File: server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ClearCachePlan.java [hunk 2]
```diff
@@ -35,4 +39,17 @@ public ClearCachePlan() {
...
+    stream.writeByte((byte) PhysicalPlanType.CLEARCACHE.ordinal());
+  }
+
+  @Override
+  public void serialize(ByteBuffer buffer) {
+    buffer.put((byte) PhysicalPlanType.CLEARCACHE.ordinal());
+  }
+
+  @Override
+  public void deserialize(ByteBuffer buffer) throws IllegalPathException {}
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ClearCachePlan.java:54]: why deserialize do not implementation？ How to deserialize this execution plan?

File: server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateSnapshotPlan.java [hunk 3]
```diff
@@ -36,4 +40,17 @@ public CreateSnapshotPlan() {
...
+    stream.writeByte((byte) PhysicalPlanType.CREATE_SNAPSHOT.ordinal());
+  }
+
+  @Override
+  public void serialize(ByteBuffer buffer) {
+    buffer.put((byte) PhysicalPlanType.CREATE_SNAPSHOT.ordinal());
+  }
+
+  @Override
+  public void deserialize(ByteBuffer buffer) throws IllegalPathException {}
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateSnapshotPlan.java:55]: why deserialize do not implementation？ How to deserialize this execution plan?

File: server/src/main/java/org/apache/iotdb/db/qp/physical/sys/MergePlan.java [hunk 4]
```diff
@@ -39,4 +43,17 @@ public MergePlan() {
...
+    stream.writeByte((byte) PhysicalPlanType.MERGE.ordinal());
+  }
+
+  @Override
+  public void serialize(ByteBuffer buffer) {
+    buffer.put((byte) PhysicalPlanType.MERGE.ordinal());
+  }
+
+  @Override
+  public void deserialize(ByteBuffer buffer) throws IllegalPathException {}
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/qp/physical/sys/MergePlan.java:58]: why deserialize do not implementation？ How to deserialize this execution plan?

File: cluster/src/test/java/org/apache/iotdb/cluster/log/applier/DataLogApplierTest.java [hunk 5]
```diff
@@ -374,4 +376,20 @@ public void testApplyCreateMultiTimeseiresWithPulling() throws MetadataException
     assertTrue(IoTDB.metaManager.getAllStorageGroupPaths().contains(new PartialPath("root.sg2")));
     assertNull(log.getException());
   }
+
```
Comment (wangchao316) [inline cluster/src/test/java/org/apache/iotdb/cluster/log/applier/DataLogApplierTest.java:379]: Good Job.

Generally, the testcontainer needs to be added to implement the functions of the cluster edition.

**Review by wangchao316 — CHANGES_REQUESTED** [review]

File: server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ClearCachePlan.java (hunk 2 above)
Comment (ijihang) [reply server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ClearCachePlan.java:54]: There are no fields to deserialize

**Review by wangchao316 — COMMENTED** [review]
hi,  if you could add some test container， if do not error,   I will approve it.

**Review by neuyilan — APPROVED** [review]
LGTM

File: cluster/src/test/java/org/apache/iotdb/cluster/log/applier/DataLogApplierTest.java (hunk 5 above)
Comment (ijihang) [reply cluster/src/test/java/org/apache/iotdb/cluster/log/applier/DataLogApplierTest.java:379]: thanks, i will add testcontainer

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**